shop-the-look/next.config.js

```diff
@@ -0,0 +1,16 @@
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  rewrites: async () => {
+    return [
+      {
```

Comment:
Perhaps note this redirect is what makes the frontend JS and Python backend work in concert.